src/gradata/hooks/brain_maintain.py

```
@@ -0,0 +1,65 @@
+            text = lessons.read_text(encoding="utf-8")
+            fts_index("lessons.md", "markdown", text)
+
+        # Index any .md files in brain root
+        for md_file in brain_path.glob("*.md"):
+            if md_file.name == "lessons.md":
+                continue
+            try:
+                text = md_file.read_text(encoding="utf-8")
+                fts_index(md_file.name, "markdown", text)
+            except Exception:
+                continue
```

Ruff (0.15.9)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)